src/main/webapp/app/scores/exercise-scores.module.ts

```diff
@@ -62,11 +63,7 @@ const ENTITY_STATES = [
         ExerciseScoresRepoExportPopupComponent,
     ],
     entryComponents: [
-        HomeComponent,
```

Comment:
These are all components from other modules and shouldn't be entryComponents of this module.

An entryComponent is needed when a component is not used in a template (that is by using it on a route or with an injector).